packages/lib/src/vis/raw/RawVis.tsx

```diff
@@ -8,23 +10,13 @@ function RawVis(props: Props) {
   const { value } = props;
   const valueAsStr = JSON.stringify(value, null, 2);
```

Comment:
This step could be time consuming if `value` is too big. I wonder if we could infer the size in another lighter way.

Reply:
Yeah, I did wonder about that. On the other hand, the `value` first has to be downloaded, which takes a _lot_ more time than `JSON.stringify`. Even in VS Code we have a 2GB limit on the size of the HDF5 file, so I'm not hugely concerned. Given that the raw vis is for datasets that have complicated dtypes, I'm not sure there's an easier way to approximate the size... So I'd prefer to wait for someone to hit this potential bottleneck before trying to fix it.

Reply:
Sounds good to me :+1: